05-AdvancedTopics/web-search-mcp/client.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
```

Comment:
I recommend adding a README.md in this folder where you describe 1) how to get an API key 2) how to run the client and the server so if they go into this folder the reader knows what to do, to get it working